readme.md

```diff
@@ -64,6 +64,12 @@ Type: `string`
 
 Capture a specific DOM element.
 
+##### css
+
+Type: `string`
+
+Apply custom CSS to the webpage.
```

Comment:
What about the description? Should we add something regarding file support and raw css support?